ios/FluentUI/Navigation/Views/LargeTitleView.swift

```diff
@@ -10,7 +10,8 @@ import UIKit
 /// Large Header and custom profile button container
 class LargeTitleView: UIView {
     enum Style: Int {
-        case light, dark
+        case primary
```

Comment:
Are these actually semantically equivalent? Otherwise, we're going to have to carefully track this when we go to update controls in devmain.

Reply:
Yep. Following the post on teams, light = primary and dark = system.